package.json

```diff
@@ -15,23 +15,30 @@
   },
   "peerDependencies": {
     "apollo-cache-inmemory": "^1.1.5",
+    "apollo-client": "^2.2.0",
     "apollo-link": "^1.0.7",
     "apollo-link-context": "^1.0.3",
     "apollo-link-error": "^1.0.3",
     "apollo-link-http": "^1.3.2",
-    "apollo-client": "^2.2.0",
-    "bs-platform": "^2.1.0"
+    "bs-platform": "^2.1.0",
+    "graphql_ppx": "^0.0.1"
```

Comment:
Silly question: it seems that the package is [released as `0.0.3`](https://github.com/mhallin/graphql_ppx/commit/365ab4fde42baaf671b5190a9500819afef64374) for ocaml but not for npm? 🤔

Also `0.0.1` hints that it's a really WIP/Experimental API. Are you willing to take the risk of things breaking or is that ok for you?

Reply:
@mhallin correct to add it as a peerDependency? 
Lot's of people want the `graphql_ppx` and me too. Today, your defining the typed result, but you could mistake in that. You could also mistake in the graphql query your writing.
With the ppx, you will be safe with your graphql query and with the generated types. (If your schema.json is up to date with your backend)

Reply:
For sure! All good, just wanted to clarify this 😉

Reply:
Graphql_ppx has 2 parts:
1. The ppx itself which is published on opam and evolve over time. Currently at version 0.0.3 on opam
2. Integration with bucklescript and npm ecosystems. It has 2 jobs:
  - Create a symlink of the ppx into node_modules so it can be pick up by bucklescript
  - Send introspection query

Reply:
From my experience with ocaml ecosystem, once a package was published , it's pretty solid